### ISSUE TITLE
Support GHC 9.2

### ISSUE THREAD

**Issue by Mikolaj** [tfausak/witch#41]
I'm not sure it's just bumping the bounds, because LambdaHack with allow-newer compiles, but crashes. I can investigate whether it's witch-caused, if that helps.

**Comment by tfausak** [comment]
Seems like it works fine with GHC 9.2: #42. Do you know what exactly crashed?

**Comment by Mikolaj** [comment]
Let me test.

**Comment by Mikolaj** [comment]
Yes, it all seems to work fine. I think the crashes I'm seeing are from sdl2: https://github.com/haskell-game/sdl2/issues/233
